mono/metadata/threads-types.h

```diff
@@ -44,6 +44,7 @@ typedef enum {
 } MonoThreadApartmentState;
 
 typedef enum {
+// These values match Windows, but are offset by 2.
```

Comment:
Unrelated to the change, this is to match the managed values, nothing to do with Win32. Remove that.

Reply:
It is both. We subtract 2 elsewhere because of this.